packages/x-components/src/design-system/components/list/default.scss

```diff
@@ -3,11 +3,11 @@
 .x-list {
   // layout
```

Comment:
Item sizing example is not working:

![image](https://user-images.githubusercontent.com/6597815/126766360-8208325b-bad3-4b80-bce0-fb6ec2570260.png)

Reply:
fixed